cmd/collector/config/config.go

```diff
@@ -107,6 +107,7 @@ type Config struct {
 	OtelLog               *OtelLog                 `toml:"otel-log,omitempty" comment:"Defines an OpenTelemetry log endpoint. Accepts OTLP/HTTP."`
 	OtelMetric            []*OtelMetric            `toml:"otel-metric,omitempty" comment:"Defines an OpenTelemetry metric endpoint. Accepts OTLP/HTTP and/or OTLP/gRPC."`
 	HostLog               []*HostLog               `toml:"host-log,omitempty" comment:"Defines a host log scraper."`
+	Exporters             *Exporters               `toml:"exporters,omitempty" comment:"Optional configuration for exporting telemetry outside of adx-mon."`
```

Comment:
Exporter is also the name that prometheus uses for components that expose metrics.  I wonder if this would be confusing.  Only other name that comes to mind is "Forwarders."

Reply:
Any OTEL uses exporter for what this is doing as well, so maybe a moot point.

Reply:
Going to stick with the otel convention for now